site/docs/src/json/applications/search-response.json

```diff
@@ -0,0 +1,161 @@
+{
+  "applications": [
+    {
+      "accessControlConfiguration": {},
+      "active": true,
+      "authenticationTokenConfiguration": {
+        "enabled": false
+      },
+      "data": {},
+      "emailConfiguration": {},
+      "externalIdentifierConfiguration": {},
+      "formConfiguration": {
+        "adminRegistrationFormId": "d8ee45a9-6782-cf31-e1e0-06f6b113a366"
+      },
+      "id": "02c0285a-18ca-4e0c-8834-e8d8e27f6fde",
+      "insertInstant": 1673296704038,
+      "jwtConfiguration": {
+        "enabled": false,
+        "refreshTokenExpirationPolicy": "Fixed",
+        "refreshTokenTimeToLiveInMinutes": 43200,
+        "refreshTokenUsagePolicy": "Reusable",
+        "timeToLiveInSeconds": 3600
+      },
+      "lambdaConfiguration": {},
+      "lastUpdateInstant": 1673296704038,
+      "loginConfiguration": {
+        "allowTokenRefresh": false,
+        "generateRefreshTokens": false,
+        "requireAuthentication": true
+      },
+      "multiFactorConfiguration": {
+        "email": {},
+        "sms": {}
+      },
+      "name": "Email and registration verification (verify email and registration)",
```

Comment:
This looks like it was taken from our dev setup. I would rather use a more likely name here such as `Forum`, `Payroll` or `Support portal`.

Reply:
👍